types/react-gallery-carousel/index.d.ts

```diff
@@ -64,12 +64,7 @@ declare namespace Carousel {
         children?: React.ReactNode;
     }
 
-    // Buttons of the carousel that accepts a boolean or string.
-    type ButtonType = boolean | string;
-
-    // Object fit css properties
-    type ValueOf<T> = T[keyof T];
-    type ObjectFit = ValueOf<Pick<React.CSSProperties, 'objectFit'>>;
+    type ObjectFit = React.CSSProperties['objectFit'];
```

Comment:
I think this is just a cleaner way of achieving the same thing as before. Please let me know if I'm wrong.